r_bc_b:
- -1.4069671580949343
- -0.1736187868617869
